packages/storybook/src/documentation/pages/pagination.stories.mdx

```diff
@@ -90,10 +107,45 @@ To achieve this, just set the pagination property of the `components` react prop
 
 <CustomPaginationDemo />
 
-## Pagination API
+## Server-side Pagination
+
+XGrid supports both client and server side pagination. By default, pagination works on the client side. To switch it to server side,
+set the property `paginationMode` to `Server` as string or you can use our `FeatureMode` enum.
+**Note**: You also need to set the `rowCount` property to override the number of rows in the grid so XGrid can calculate the total number of pages.
+
+Finally, you're going to need to handle the `onPageChanged` event, to load the rows for the corresponding page.
+
+```tsx
+const [rows, setRows] = useState<RowsProp>([]);
+const onPageChanged = useCallback(
+  (params) => {
+    loadServerRows(params).then((newRows) => {
+      setRows(newRows);
+    });
+  },
+  [setRows],
+);
+
+<XGrid
+  rows={rows}
+  columns={columns}
+  options={{
+    pagination: true,
+    pageSize: 5,
+    paginationMode: 'Server',
```

Comment:
This looks off, either it should be lowercase or importing a constant variable.
```suggestion
    paginationMode: 'server',
```

Reply:
Why not in pascal case?
You can pass a string or an enum, `FeatureMode.Server` which equal to `Server`

Reply:
It would be inconsistent. So far, we either use lower cases for enums props or upper case for constants. I don't that introducing a third casing strategy is an option.

Reply:
ok let's go lower case then